pynet/preprocessing/spatial.py

```diff
@@ -116,8 +118,43 @@ def scale(im, scale, tmpdir=None):
             normalized.get_data(), normalized.affine)
     return normalized
 
+def bet2(im, f=0.5, tmpdir=None, pkg_version=0):
+    """ Skull stripped the MRI image.
 
-def reorient2std(im, tmpdir=None):
+    This function is based on FSL.
+
+    Parameters
+    ----------
+    im: nibabel.Nifti1Image
+        the input image.
+    tmpdir: str, default None
+        a folder where the intermediate results are saved.
+    f: float, default 0.5
+        -fractional intensity threshold (0->1);smaller values give larger brain outline estimates
+    pkg_version : int, default 0
+        put to 1 if the package is not installed with the source repository
+
+    Returns
+    -------
+    skullstripped: nibabel.Nifti1Image
+        the skull stripped input image.
+    """
+    check_version("fsl",pkg_version)
+    check_command("bet")
+    with TemporaryDirectory(dir=tmpdir, name="skullstripped") as tmpdir:
+        input_file = os.path.join(tmpdir, "input.nii.gz")
+        output_file = os.path.join(tmpdir, "output.nii.gz")
+        nibabel.save(im, input_file)
+        cmd = ["bet", input_file, output_file, "-f", str(f)]
+        logger.debug(" ".join(cmd))
+        subprocess.check_call(
+            cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
- we discussed this and decided to remove unnecessary PIPE.

Reply:
I used Popen and communicate to see more details when it failed.